BuildLoader.py

```diff
@@ -1089,10 +1089,15 @@ def pre_build(self):
             raise Exception  ('IPP_HASH_LIB_SUPPORTED_MASK is not set correctly!!')
 
         # check if FSP binary exists
-        fsp_dir  = os.path.join(plt_dir, 'Silicon', self._board.SILICON_PKG_NAME, "FspBin", self._board._FSP_PATH_NAME)
+        if self._board._FSP_PATH_NAME != '':
+            fsp_path_name = self._board._FSP_PATH_NAME
+        else:
+            fsp_path_name = os.path.join('Silicon', self._board.SILICON_PKG_NAME, "FspBin", self._board._FSP_PATH_NAME)

```

Comment:
line 1095, you don't need to add self._board._FSP_PATH_NAME part since it is empty already.

Reply:
Agree. Let me remove it.